addOns/reports/CHANGELOG.md

```diff
@@ -5,9 +5,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
-### Added
+### Added 
+- The Traditional JSON report, Traditional JSON Report with requests and responses, Traditional XML Report and Traditional XML Report with requests and responses now has "otherinfo" field per alert (Issue 7260).
+
 - A description of riskdesc fields in the relevant report templates' help (Issue 7445).
 
+### Deprecated
+- The old "otherinfo" entry to be phased out.
+
+### Added
+
```

Comment:
The heading and one blank line should be removed.

Reply:
yes sorry